### ISSUE TITLE
Suggestion:  Filelist supports environment variables without any parentheses

### ISSUE THREAD

**Issue by jacklee023** [dalance/svlint#134]
Thank you for creating such a great Linter tool !

I'd like to make a suggestion. Can you support environment variables without parentheses? 
Like this format:
```
$XXX_DIR/yyy.sv
```
This format comes up so often in my projects that I can't fix them all

I think this is an easy change and would like to update the code to support it
Thank you again for your work

**Comment by DaveMcEwan** [comment]
This issue should be raised on [verilog-filelist-parser](https://github.com/raamakrishnan/verilog-filelist-parser) instead of svlint.
@dalance Would you close this issue?
